src/lib/containers/TermsAndConditions.tsx

```diff
@@ -94,10 +95,13 @@ Please refer to our full <a target="_blank" href="https://help.synapse.org/docs/
 
 const TermsAndConditions: React.FunctionComponent<TermsAndConditionsProps> = ({
   onFormChange,
+  showTcTerm = true,
```

Comment:
1. Not a great name, how about `showLinkToFullTC`?
2. We haven't been consistent about this in this codebase, but it is often better to make optional boolean props default to false. If the default behaviour here should be to show, then switch `show` to `hide` and make the default `false`. Then when rendering the component in JSX, if you wanted `hideLinkToFullTC` to be true, you can just write:

```jsx
<TermsAndConditions hideLinkToFullTC />
```

```suggestion
  hideLinkToFullTC = false,
```